src/execution_plan/ops/op_project.h

```diff
@@ -20,6 +20,7 @@ typedef struct {
     bool singleResponse;            // When no child operations, return NULL after a first response.
     unsigned short exp_count;       // Number of projected expressions.
     unsigned short order_exp_count; // Number of order by expressions.
+    uint record_len;
```

Comment:
Missing comment, explaining the purpose of this field.

Reply:
done